cmd/keymasterd/config.go

```diff
@@ -204,6 +227,39 @@ func loadVerifyConfigFile(configFilename string) (RuntimeState, error) {
 		logger.Debugf(3, "client ca file loaded")
 
 	}
+	if len(runtimeState.Config.Base.KeymasterPublicKeysFilename) > 0 {
+		filename := runtimeState.Config.Base.KeymasterPublicKeysFilename
+		if _, err := os.Stat(filename); os.IsNotExist(err) {
+			logger.Printf("keymaster_public_keys_filename defined but file does not exist")
+			return runtimeState, err
+		}
+		inFile, err := os.Open(filename)
+		if err != nil {
+			logger.Printf("keymaster_public_keys_filename cannot be opened")
+			return runtimeState, err
+		}
+		defer inFile.Close()
+		scanner := bufio.NewScanner(inFile)
+		scanner.Split(bufio.ScanLines)
+		for scanner.Scan() {
+			logger.Debugf(2, "line='%s'", scanner.Text())
+			userPubKey := scanner.Text()
+			sshPubKey, _, _, _, err := ssh.ParseAuthorizedKey([]byte(userPubKey))
+			if err != nil {
+				return runtimeState, err
+			}
+			//
+			cryptokey, ok := sshPubKey.(ssh.CryptoPublicKey)
+			if !ok {
+				err := errors.New("cannot cast public key!")
+				return runtimeState, err
```

Comment:
You could collapse the `err` assignment and the `return` onto one line (skipping the assignment).